test/zdtm.py

```diff
@@ -2738,7 +2738,17 @@ def clean_stuff(opts):
 for tst in test_classes.values():
     tst.available()
 
+orig_hugepages = ""
+with open("/proc/sys/vm/nr_hugepages", "r") as f:
+    orig_hugepages = f.read()
+
+with open("/proc/sys/vm/nr_hugepages", "w") as f:
+    f.write("20\n")
+
 opts['action'](opts)
 
+with open("/proc/sys/vm/nr_hugepages", "w") as f:
+    f.write(orig_hugepages)
```

Comment:
Would it be possible to move these into functions, for example, `set_nr_hugepages()` and `restore_nr_hugepages()`?